config/configfile.yaml

```diff
@@ -9,7 +9,7 @@ exclude: "config/outliers.txt"
 description: "config/description.md"
 
 strain_id_field: "accession"
-display_strain_field: "strain_original"
+display_strain_field: "strain"
```

Comment:
To rescue `display_strain_field`, need to change

https://github.com/nextstrain/rsv/blob/e012512b0e35d1c1f575bed78eae48c5ae6f8920/scripts/set_final_strain_name.py#L27

so that it takes a `--metadata-id-columns` flag